ur_simulation_ignition/launch/ur_sim_moveit.launch.py

```diff
@@ -67,10 +67,11 @@ def launch_setup(context, *args, **kwargs):
 
     ur_moveit_launch = IncludeLaunchDescription(
         PythonLaunchDescriptionSource(
-            [FindPackageShare("ur_moveit_config"), "/launch", "/ur_moveit.launch.py"]
+            [FindPackageShare("ur_simulation_ignition"), "/launch", "/ur_moveit.launch.py"]
```

Comment:
Please test with the above-mentioned branch.

Reply:
Has been tested, see other comment.